releases/v1.2.0.toml

```diff
@@ -87,6 +87,10 @@ Add support for cleaning up leases and content ingests to garbage collections.
 
 Add expiration label to clean up temporary resources.
 
+## Image Importer
+
+The image importer has been updated to support output from `docker save`
```

Comment:
Maybe we can also mention `ctr images import` CLI incompatibility, although it is out of scope of the support.